mkdocs.yml

```diff
@@ -61,6 +61,7 @@ nav:
       - ROS 2 Network Config: setup/xml-config.md
       - Setup Compute Boards:
         - ROS 2 on Ubuntu 20.04: setup/ubuntu2004.md
+        - ROS 2 on a NavQ+: setup/navqplus.md
```

Comment:
To mirror the setup on the hookup examples. please put this after the NVIDIA line. I'll take it as an action that we need a better way to highlight community-provided content versus content we are responsible for. Probably also I should rework this section to divide setup of specific hardware versus more generic software setup.

Reply:
No need to move this for this PR; I will rework the yaml after it merges.